sdk/keyvault/keyvault-secrets/src/index.ts

```diff
@@ -26,14 +26,36 @@ import {
   SetSecretOptions,
   UpdateSecretOptions,
   GetSecretOptions,
-  GetAllSecretsOptions,
+  GetSecretsOptions,
   SecretAttributes
 } from "./secretsModels";
 import { parseKeyvaultIdentifier as parseKeyvaultEntityIdentifier } from "./core/utils";
 import { NewPipelineOptions, isNewPipelineOptions, Pipeline } from "./core/keyVaultBase";
-import { TelemetryOptions } from "./core";
+import { ProxyOptions, RetryOptions, TelemetryOptions, ParsedKeyVaultEntityIdentifier } from "./core";
```

Comment:
Maybe a silly question..., but do we **have** to import something to the `index.ts` file in order to just export it? I recall the `index.ts` files in event hubs and service bus exporting directly. 

See https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/src/index.ts

Reply:
For some of the definitions, where I don't use them in index.ts, you're right, I could have just exported them. Here, I took a slightly different style and grouped my imports by module.